sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
 # Release History
 
 ## 1.0.0-beta.1 (Unreleased)
+
+This is the first preview of the Azure.AI.Translation.Document client library that targets the service version `1.0-preview.1`.
```

Comment:
are we saying "beta" now instead of "preview"? I thought so, but wasn't sure.

Reply:
Good catch. I will update